cmd/lotus-shed/send-csv.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/ipfs/go-cid"
+	"github.com/urfave/cli/v2"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/exitcode"
+
+	lapi "github.com/filecoin-project/lotus/api"
+	"github.com/filecoin-project/lotus/chain/types"
+	lcli "github.com/filecoin-project/lotus/cli"
+)
+
+var sendCsvCmd = &cli.Command{
+	Name:      "send-csv",
+	Usage:     "Utility for sending a batch of balance transfers",
+	ArgsUsage: "[sender] [csvfile]",
+	Action: func(cctx *cli.Context) error {
+		if cctx.NArg() != 2 {
```

Comment:
I would personally use the `--from` flag instead of a positional argument, its standard across a lot of other things in our tooling

Reply:
fine, but are you okay with it being mandatory? i hate "default wallet" more with each passing day, especially for something like this.